opentelemetry-zipkin/src/exporter/mod.rs

```diff
@@ -106,6 +106,7 @@ impl ZipkinPipelineBuilder {
                 ));
                 cfg
             } else {
+                #[allow(deprecated)]
```

Comment:
Would be good to cleanup this instead of having deprecated code. Not a blocker for this PR though :)

Reply:
it seems like Zipkin needs to be updated to follow the builder pattern that was added for otlp also, its currently using the pipeline version.

I will try and get this refactored to not use the allow.

Reply:
maybe there can be a followup for the zipkin to match the otlp crate.

Reply:
Its best to do as a follow up to keep the scope for this PR smaller.

Reply:
yup, that's what I was thinking. I went simple with the approach on this PR, but I would like to open another to move to the Builder pattern if that would be welcomed

Reply:
my current change actually ignores the config, I'm going to revert to using the deprecated annotation, and then do a followup PR to move to the builder pattern to avoid too many changes in this PR.